Scrumdinger/Models/DailyScrum.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  DailyScrum.swift
+//  Scrumdinger
+//
+//  Created by Life Boss on 7/5/23.
+//
+
+import Foundation
+
+struct DailyScrum: Identifiable {
+    let id: UUID
+    var title: String
+    var attendees: [String]
+    var lengthInMinutes: Int
+    var theme: Theme
+    
+    init(id: UUID = UUID(), title: String, attendees: [String], lengthInMinutes: Int, theme: Theme) {
```

Comment:
☝️ By assigning a default value here, we don't need to specifically define the `id` when constructing each `DailyScrum` object.